scripts/vim-patch.sh

```diff
@@ -125,8 +125,12 @@ commit_message() {
 }
 
 find_git_remote() {
-  git_remote=$(git remote -v \
-    | awk '$2 ~ /github.com[:\/]neovim\/neovim/ && $3 == "(fetch)" {print $1; exit}')
+  local git_remote
+  if [[ "$1" == fork ]]; then
```

Comment:
./scripts/vim-patch.sh: line 129: $1: unbound variable

Reply:
I changed it to `${1-}`.